core/arch/arm/plat-k3/drivers/sa2ul.c

```diff
@@ -116,10 +117,15 @@ static TEE_Result sa2ul_init(void)
 
 	/* Modify TRNG firewall to block all others access */
 	control = FW_ENABLE_REGION;
-	permissions[0] = (FW_BIG_ARM_PRIVID << 16) | FW_SECURE_ONLY;
 	start_address = RNG_BASE;
 	end_address = RNG_BASE + RNG_REG_SIZE - 1;
+	permissions[0] = (FW_BIG_ARM_PRIVID << 16) | FW_SECURE_ONLY;
+#if defined(PLATFORM_FLAVOR_am62x)
+	permissions[1] = (FW_TIFS_PRIVID << 16) | FW_NON_SECURE;
+	ret = ti_sci_set_fwl_region(fwl_id, rng_region, 2,
```

Comment:
A stylistic comment but we should not split the function call like this when we can create a variable `num_permissions` and that might look cleaner.

Reply:
+1